src/patternfly/components/ModalBox/modal-box-title-icon.hbs

```diff
@@ -0,0 +1,20 @@
+<span class="pf-c-modal-box__title-icon{{#if modal-box-title-icon--modifier}} {{modal-box-title-icon--modifier}}{{/if}}"
+  {{#if modal-box-title-icon--attribute}}
+    {{{modal-box-title-icon--attribute}}}
+  {{/if}}>
+  {{#if modal-box--IsDefaultAlert}}
+    <i class="fas fa-bell" aria-hidden="true"></i>
+  {{else if modal-box--IsInfoAlert}}
+    <i class="fas fa-info-circle" aria-hidden="true"></i>
+  {{else if modal-box--IsSuccessAlert}}
+    <i class="fas fa-check-circle" aria-hidden="true"></i>
+  {{else if modal-box--IsWarningAlert}}
```

Comment:
should this use fa-fw so all the icons line up?

Reply:
updated!